### DISCUSSION TITLE
How to use translations?

### DISCUSSION THREAD

**Discussion by Defcon0** [diasurgical/DevilutionX#2946] [Q&A] [answered]
Hello,

might be obvious, but how do I use the translation *.po files provided in the Translations folder in this repo? I tried putting the de.po in my devilutionx folder and setting 

```
[Language] 
Code=de
```

in my diablo.ini

But without success.

What am I doing wrong?

Thanks!

Bye Defcon0

**Comment by AJenbo** [comment]
See the instruction for testing translations here: https://github.com/diasurgical/devilutionX/wiki/Translation-Guide#step-3-testing

Would you be interested in helping with finishing the German translation?

**Comment by Defcon0** [comment, marked as answer]
Sorry, due to work-related time constraints I can't atm. But I keep it in mind. Thanks for your awesome work!! I just discovered devilutionx and already love it! Only the auto clicker I didn't achieve, yet. I tried the one from the "fresh meat" fansite (diablo clicker v3) but it didn't work. Seems to be done only for the vanilla diablo.exe :-(

**Reply by AJenbo** [reply]
Sounds like you are not using the latest test build, in that case translations won't show either as both of these features are for the upcoming 1.3.0 release (coming this October). German is currently 83% translated.

**Reply by Defcon0** [reply]
You're right, I use the current stable release. Thrilled to hear, the features (83% is awesome already) will also be available in the next stable one. I'll take a look when it's ready 👍

**Comment by Trejanus** [comment]
You are doing a great thing! Thanks to you, Diablo Hellfire is still alive. I am waiting for version 1.3 with Polish translation. I hope it will work :-)

**Reply by AJenbo** [reply]
Working on integrating the Polish dub right now :)

**Comment by Trejanus** [comment]
Excellent! But good news! 🥇 👍 
What time perspectives (because I understand that it will be in version 1.3)? 
Another technical question, how will the language selection work? Somewhere in vogue or before running it in an .ini file?
:-)

**Reply by AJenbo** [reply]
Next week

**Reply by Trejanus** [reply]
Hura!!!!! 🥇 
:-)

**Comment by Trejanus** [comment]
I will also ask if you can contact WiA Parker or Noktis (fan version of the Diablo localization) :-)

**Reply by qndel** [reply]
We contacted both already, Noktis seems busy at the moment but there's no reason why we can't have two polish translations later if he decides to put time into it and some people think the difference matters and would choose his one.

**Reply by Trejanus** [reply]
I support you very much and I wish you success for your project. This is a great thing. I think that with such an attitude, your mod will be a must for everyone who wants to play good old Diablo / Hellfire :-) 

I wish you fruitful cooperation with WiA and Noktis. They should support your mod with all their might for the sake of the iconic game and fans ...

**Comment by Trejanus** [comment]
All Diablo fans from Poland are waiting for the Polish version of the mod. Personally, I am looking forward to it :-)
Full support for Your great work! :-)

**Comment by AJenbo** [comment]
1.3.0 is now available from Google Play, if your phone is set to Polish as it's default language, then it will automatically download the translation on first run.

Instructions for other platforms, and how to force it even if your phone has a different language will be made available later this week

**Comment by Trejanus** [comment]
Otherwise, I have a PC version of Diablo (GOG version). I have installed the DevilutionX 1.2.1 mod. How to download 1.3.0? Wait ? You still have 1.2.1 on the website and ... Adnroid on Google Play. I need a PC version, not an Android version. 
:-)

**Reply by AJenbo** [reply]
The Android version of 1.3.0 ended up being released a few days ahead the other port, since it's a new platform and a different release channel things are a bit different there. I'm still in the process of writing release notes for the other versions. Once that is done (expected Sunday) 1.3.0 will be available for everyone.

**Comment by Trejanus** [comment]
So I keep my fingers crossed. I have to wait for the PC port. Thanks for your work. This is a great thing :-)

**Comment by Trejanus** [comment]
Thanks for ver 1.3.0 and Polish translation !!!! Poland thanks and is grateful :-D  👍 🥇
